test/test_cuda.py

```diff
@@ -2784,7 +2784,7 @@ def test_triangular_solve(self):
         _TestTorchMixin._test_triangular_solve(self, lambda t: t.cuda())
 
     @unittest.skipIf(not TEST_MAGMA, "no MAGMA library detected")
-    @unittest.skip("Spuriously failing")
+    @skipCUDANonDefaultStreamIf(True)
```

Comment:
If we're going to skip this we should put a message as to why (i.e., tringular_solve doesn't work with non default stream, or there is something wrong with the non-default stream testing) and link to an issue.

Reply:
I believe triangular_solve doesn't work with non-default stream. This is an issue that takes this generally.

Reply:
`test_triangular_solve` is also skipped in an analogous manner.